Tests/UITests/TestApp/LLMOpenAI/Onboarding/LLMOpenAIModelOnboarding.swift

```diff
@@ -13,11 +13,24 @@ import SwiftUI
 
 struct LLMOpenAIModelOnboarding: View {
     @Environment(OnboardingNavigationPath.self) private var path
-
+    @State private var showingAlert = false
+    @State private var modelSelection = ""
     
     var body: some View {
-        LLMOpenAIModelOnboardingStep { _ in
-            path.removeLast()
+        Group {
+            LLMOpenAIModelOnboardingStep { model in
+                modelSelection = model
+                self.showingAlert.toggle()
+            }
+        }
+        .alert(isPresented: $showingAlert) {
```

Comment:
Is the `Alert` really necessary? I guess you only need it for the respective UI test? Maybe a small `Text` element somewhere would be more suitable?

Reply:
The action button both updates the result and navigates away immediately, so the alert provides a pause such that the UI test can see the text before navigating away - is there a better solution?